test/functionalTest/cases/0000_deprecated_checkings/get_entity_dates_with_options.test

```diff
@@ -49,7 +49,7 @@ payload='{
   "type": "T",
   "A": "foo"
 }'
-T_0=$(TZ=GMT date +"%Y-%m-%dT%H:%M:%S.00Z")
+T_0=$(TZ=GMT date +"%Y-%m-%dT%H:%M:%S")
```

Comment:
I understand `date` command is not able to provide date with miliseconds precission, isn't it?

Reply:
I don't know. Didn't check that - didn't even think about it to tell you the truth . ..

Reply:
`man date`:
%N     nanoseconds (000000000..999999999)

However, it doesn't help.
If we get nanoseconds we still can't compare with the milliseconds we now get from the broker.

NTC